app/components/Card.tsx

```diff
@@ -3,17 +3,19 @@ import type { PropsWithChildren } from 'react'
 
 export type CardProps = {
   isSelected?: boolean,
-  onTileClicked?: () => void
+  onTileClick?: () => void,
+  classes?: string
```

Comment:
I'm still not sold on the idea of passing through the classname here but regardless:
this should be called `className` instead of `classes`